### DISCUSSION TITLE
Question - Webpack Module Federation - Angular & Flutter Web

### DISCUSSION THREAD

**Discussion by AlvinaC** [webpack/webpack#15280] [Q&A]
Hi,

I have an angular application which works independently. Also there is a flutter web build for a certain feature which runs independently. I would like to use the flutter web app as a micro frontend which can be viewed in angular shell application using angular routes. Does the Webpack plugin support flutter apps?

Thanks a lot.

**Comment by alexander-akait** [comment]
Sorry, better ask this in stack overflow

**Comment by dJani97** [comment]
> I would like to use the flutter web app as a micro frontend which can be viewed in angular shell application using angular routes

@AlvinaC I want to do the same. Did you find a solution?

**Edit:** I found your [unanswered question](https://stackoverflow.com/questions/70922711/does-webpack-plugin-for-microfrontends-support-flutter-web-app-be-used-as-microf). In case anyone else is interested, please go ahead and upvote it.
